src/rpc/server.h

```diff
@@ -198,6 +199,8 @@ extern std::string HelpRequiringPassphrase();
 extern std::string HelpExampleCli(const std::string& methodname, const std::string& args);
 extern std::string HelpExampleRpc(const std::string& methodname, const std::string& args);
 
+extern void TxToJSON(const CTransaction& tx, const uint256 hashBlock, UniValue& entry);
```

Comment:
Hm, I think you can simply remove the changes to this file as the function is in core_write now?